packages/test/test-service-load/src/nodeStressTest.ts

```diff
@@ -97,10 +97,19 @@ async function orchestratorProcess(
         undefined,
         args.browserAuth);
 
-    // Create a new file if a testId wasn't provided
-    const url = args.testId !== undefined
-        ? await testDriver.createContainerUrl(args.testId)
-        : await initialize(testDriver, seed, profile, args.verbose === true);
+    // If testId is provided, then try to load the file first.
+    let url;
+    if (args.testId !== undefined) {
+        const docExists: boolean = await testDriver.doesDocumentExists(args.testId);
```

Comment:
We run this test against tinylicious and routerlicious on all commits to `main`.  This looks like it will break those test runs since it's NYI for those drivers.  Should be easy to move your old implementation from earlier in the PR into those drivers too (call createContainerUrl and catch if not exists).  If you think it's worth following up with a better implementation later, then open an issue.

Alternate idea - make the script params more explicit rather than this implicit overloading.